src/calendar/actions/calendar-event/get-all-events-for-calendar-by-owner.ts

```diff
@@ -0,0 +1,30 @@
+import { Fetcher } from '../../../common/api';
+import { endpoints } from '../../api/calendar-event/endpoints';
+import { ICalendarEvent } from '../../entities';
+import { ISearchResult } from '../../../common';
+import { ICalendarEventFilterDto } from '../../types';
+import { errorFactory, CalendarErr } from '../../errors/errors';
+
+export const getCalendarEventForOwner = async (
```

Comment:
```suggestion
export const getCalendarEventsForOwner = async (
```

On peut mettre au pluriel comme on va récupérer une liste d'events.